test/e2e/storage/testsuites/multivolume.go

```diff
@@ -503,3 +527,62 @@ func TestConcurrentAccessToSingleVolume(f *framework.Framework, cs clientset.Int
 		utils.CheckReadFromPath(f, pod, *pvc.Spec.VolumeMode, path, byteLen, seed)
 	}
 }
+
+// getCurrentTopologies() goes through all Nodes and returns unique driver topologies and count of Nodes per topology
+func getCurrentTopologiesNumber(cs clientset.Interface, keys []string) ([]topology, []int, error) {
+	nodes, err := e2enode.GetReadySchedulableNodes(cs)
```

Comment:
This is already called earlier. Can we pass in the nodes instead of calling it again?

Reply:
Good point. Will change that.